tools/gn

```diff
@@ -236,13 +236,20 @@ def to_gn_args(args):
       gn_args['target_cpu'] = args.windows_cpu
     else:
         # Building host artifacts
-        gn_args['target_cpu'] = 'x64'
+        if sys.platform == 'win32':
```

Comment:
The poorly documented convention in this script is to require the OS to be explicitly specified when doing a cross-build. I think we should continue following that, or at least not fix it just now. That means that this would stay as:
```
gn_args['target_cpu'] = 'x64'
```

and you'd always pass `--windows` and `--windows-cpu`, which means we'd need to add an explicit `--windows` flag.

Reply:
> The poorly documented convention in this script is to require the OS to be explicitly specified when doing a cross-build. I think we should continue following that, or at least not fix it just now. That means that this would stay as:
> 
> ```
> gn_args['target_cpu'] = 'x64'
> ```
> 
> and you'd always pass `--windows` and `--windows-cpu`, which means we'd need to add an explicit `--windows` flag.

Done! Thanks for advise. @zanderso  Now using `--windows` and the out dir will be `win_debug_x86`.